docs/public/_redirects

```diff
@@ -1 +1,2 @@
 /:lang/*    /:lang/404/   404
+/ph/ https://astro-houston-ph.pages.dev/ 200
```

Comment:
I suspect we need the opposite order here. Netlify processes these top to bottom so the `/:lang/*` pattern is matching `/ph/` first.

```suggestion
/ph/        https://astro-houston-ph.pages.dev/   200
/:lang/*    /:lang/404/   404
```

Reply:
Ahh, this makes a lot of sense :sweat_smile: